letsencrypt-win-simple/Plugin/ManualPlugin.cs

```diff
@@ -107,7 +107,14 @@ public override void HandleMenuResponse(string response, List<Target> targets)
                 if (Program.Options.San)
                 {
                     Console.Write("Enter all Alternative Names seperated by a comma ");
-                    Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
+
+                    //Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
```

Comment:
I will fix up the patch and do another pull request.

In letsencrypt-win-simple/Plugin/ManualPlugin.cs
<https://github.com/Lone-Coder/letsencrypt-win-simple/pull/196#discussion_r59407793>
:

> @@ -107,7 +107,14 @@ public override void HandleMenuResponse(string response, List<Target> targets)
>                  if (Program.Options.San)
>                  {
>                      Console.Write("Enter all Alternative Names seperated by a comma ");
> -                    Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
> +
> +                    //Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));

You don't need to leave the old code in there since you are replacing it.

Otherwise it looks and works great. It accepted up to 16382 characters for
me. That's up to 162 characters per domain up to the 100 domain limit. If
someone goes over that, I think they need shorter domains.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/Lone-Coder/letsencrypt-win-simple/pull/196/files/cba66d56dfb3ad646d7fbdde229aa00891f4b326#r59407793>